pkg/gui/password.go

```diff
@@ -18,8 +18,9 @@ func AskPasswordDeleteWallet(nickname string, app *fyne.App) string {
 // inspired by https://hackernoon.com/asyncawait-in-golang-an-introductory-guide-ol1e34sg
 
 // Thyra password input dialog.
-func PasswordDialog(nickname string, app *fyne.App) chan string {
+func PasswordDialog(nickname string, app *fyne.App) (chan string, chan bool) {
```

Comment:
Why did you prefer to use a boom instead of a string pointer if the string is optional ?

Reply:
Because cancel and empty password produce the same result. 

which is passwordText is an empty string and to catch the difference between the two action, i had to create a "trigger" or a "flag" to catch it on the other side. Then i used adding a bool. 

But if you think there is a better way with less impact. Let's change it